backend/compact-connect/lambdas/python/common/cc_common/data_model/schema/provider/api.py

```diff
@@ -73,3 +76,48 @@ class ProviderGeneralResponseSchema(ForgivingSchema):
     homeJurisdictionSelection = Nested(
         ProviderHomeJurisdictionSelectionGeneralResponseSchema(), required=False, allow_none=False
     )
+
+
+class ProviderPublicResponseSchema(ForgivingSchema):
+    """
+    Provider object fields that are sanitized for the public lookup endpoints.
+
+    This schema is intended to be used to filter from the database in order to remove all fields not defined here.
+    It should NEVER be used to load data into the database. Use the ProviderRecordSchema for that.
+
+    This schema should be used by any endpoint that returns provider information to the public lookup endpoints
+    (ie the public query provider and public GET provider endpoints).
+
+    Serialization direction:
+    Python -> load() -> API
+    """
+
+    providerId = Raw(required=True, allow_none=False)
+    type = String(required=True, allow_none=False)
+
+    dateOfUpdate = Raw(required=True, allow_none=False)
+    compact = Compact(required=True, allow_none=False)
+    licenseJurisdiction = Jurisdiction(required=True, allow_none=False)
+    npi = NationalProviderIdentifier(required=False, allow_none=False)
+    licenseNumber = String(required=False, allow_none=False, validate=Length(1, 100))
+    licenseType = String(required=True, allow_none=False)
+    jurisdictionStatus = ActiveInactive(required=True, allow_none=False)
+    givenName = String(required=True, allow_none=False, validate=Length(1, 100))
+    middleName = String(required=False, allow_none=False, validate=Length(1, 100))
+    familyName = String(required=True, allow_none=False, validate=Length(1, 100))
+    suffix = String(required=False, allow_none=False, validate=Length(1, 100))
+    # This date is determined by the license records uploaded by a state
+    # they do not include a timestamp, so we use the Date field type
+    dateOfExpiration = Raw(required=True, allow_none=False)
+    status = ActiveInactive(required=True, allow_none=False)
+
+    privilegeJurisdictions = Set(String, required=False, allow_none=False, load_default=set())
+    providerDateOfUpdate = Raw(required=False, allow_none=False)
+
+    # these records are present when getting provider information from the GET endpoint
+    # so we check for them here and sanitize them if they are present
+    licenses = List(Nested(LicensePublicResponseSchema(), required=False, allow_none=False))
+    privileges = List(Nested(PrivilegePublicResponseSchema(), required=False, allow_none=False))
+    homeJurisdictionSelection = Nested(
```

Comment:
Do we want to include this? We're explicitly telling the caller whether or not this user is already registered in the system if we provide this value.